src/extension.ts

```diff
@@ -187,7 +187,7 @@ function callWsdl2Rest(wsdl2restExecutablePath: string): Promise<boolean> {
 
 			let outPath: string = path.join(storagePath, outputDirectory);
 			
-			if (!(wsdlFileUri.startsWith('http:') || !wsdlFileUri.startsWith('https:'))) {
+			if (!((wsdlFileUri.startsWith('http:') || wsdlFileUri.startsWith('https:')))) {
```

Comment:
the added parentheses are useless. The issue was the !

(SonarLint is reporting the useless parentheses)